src/include/libutil.h

```diff
@@ -50,6 +50,8 @@ extern "C" {
 
 /* misc_utils specific */
 
+#define IS_EMPTY(str) (str && str[0] != '\0')
```

Comment:
Shouldn't this be (!str || str[0] == '\0') ? right now, IS_EMPTY(NULL) and IS_EMPTY("\0") will return false